local/network.go

```diff
@@ -1181,12 +1179,16 @@ func (ln *localNetwork) buildArgs(
 	flags := map[string]string{
 		config.NetworkNameKey: fmt.Sprintf("%d", ln.networkID),
 		config.DataDirKey:     dataDir,
-		config.DBPathKey:      dbDir,
-		config.LogsDirKey:     logsDir,
 		config.PublicIPKey:    publicIP,
 		config.HTTPPortKey:    fmt.Sprintf("%d", apiPort),
 		config.StakingPortKey: fmt.Sprintf("%d", p2pPort),
 	}
+	if dbDir != filepath.Join(dataDir, defaultDBSubdir) {
```

Comment:
not clear for me if we need this.we can  just check if dbDir is not ""

Reply:
dbDir is not going to be empty from previous logic. the check here is to avoid adding a flag if it is
the default avalanchego value.